### PR TITLE
Home Page

### PR DESCRIPTION
**Pull request by foolhardy21**
This PR has:
- Home Page
- Currently the category/brand links don't work as I have not merged other branches yet.

Feel free to review. Thanks.

### CODE REVIEW

**Review by aayushjain-code — COMMENTED** [review]
1. You can make search box text and icon more center align.
2.Try to add some padding left and right in nav-bar

File: home.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            <p class="pos-absolute top-right-1 c-white font-size-md">converse</p>
+        </article>
+        
+        <article class="brand-card pos-relative">
+            <img src="./assets/nb.jpg" class='card-image' />
+            <p class="pos-absolute top-right-1 c-white font-size-md">new balance</p>
+        </article>
+        
+        <article class="brand-card pos-relative">
+            <img src="./assets/nike.jpg" class='card-image' />
```
Comment (anik31) [inline home.html]: add alt to img

**Comment by anik31** [comment]
center align the sneaker store text

**Comment by foolhardy21** [comment]
thanks @Aayushjain-code and @anik31 . I have made the changes.

File: home.html (hunk 1 above)
Comment (foolhardy21) [reply home.html]: I have added alt attribute to images.

**Comment by foolhardy21** [comment]
suggested changes done. merging now.

**Pull request merged**